Makefile

```diff
@@ -18,15 +18,15 @@ test:
 
 build-services:
 	@ echo -e '$(BUILD_PRINT)Building the containers'
-	@ docker-compose --file docker/docker-compose.yml --env-file docker/.env build
+	@ docker-compose --file docker/docker-compose.yml --env-file .env build
```

Comment:
I believe `docker-compose v1.25` treats the `--env-file` parameter relative to the CWD, while `docker-compose v1.28` treats the `--env-file` parameter relative to the docker-compose.yml file. 

This is probably the reason for the GitHub action failing.